features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootView.kt

```diff
@@ -175,7 +182,14 @@ fun CreateRoomSearchBar(
         },
         shape = if (!active) SearchBarDefaults.dockedShape else SearchBarDefaults.fullScreenShape,
         colors = if (!active) SearchBarDefaults.colors() else SearchBarDefaults.colors(containerColor = Color.Transparent),
-        content = {},
+        content = {
+            results.forEach {
```

Comment:
Do we have a known limit on the search? I wonder if we should use `LazyColumn` instead?